ui/.prettierignore

```diff
@@ -0,0 +1 @@
+pnpm-lock.yaml
```

Comment:
I'm surprised that this needs to be ignored explicitly... doesn't `prettier` only look at JS / TS code files? If not (by default), would it instead make sense to configure it to only look at JS / TS code files?

Reply:
I guess https://prettier.io/docs/en/options.html#parser is the proper reference.

Reply:
[Prettier supports other languages as well](https://prettier.io/docs/en/), and for us it IMO makes sens to also format at least the JSON files in the UI directory to, as there's currently no formatter for JSON. If the project as a whole (on root level) starts using some other formatted for other files than TS/JS, it would probably then make sense to use prettier only for TS/JS, or what do you think?

Reply:
Yeah, I was concerned that running prettier on the whole directory recursively would be overly broad. But it really seems like we'd want *all* supported files in the directory to get formatted, so let' keep it as-is.